maplibre/src/coords.rs

```diff
@@ -48,14 +48,65 @@ impl Quadkey {
 
 impl fmt::Debug for Quadkey {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let len = self.0[0] as usize;
+        let len = self.0[0].0 as usize;
         for part in &self.0[0..len] {
             write!(f, "{:?}", part)?;
         }
         Ok(())
     }
 }
 
+#[derive(Ord, PartialOrd, Eq, PartialEq, Hash, Copy, Clone, Debug)]
+pub struct ZoomLevel(u8);
+
+impl ZoomLevel {
+    pub fn new(zoom_level: u8) -> Self {
+        ZoomLevel(zoom_level)
+    }
```

Comment:
Should probably be removed becausee we have the `From` trait.

Reply:
Yep. You are right. Deleted.